spec/requests/members_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+describe 'Members', type: :request do
+  let(:body) { JSON.parse(response.body) }
+  let(:headers) { { "Accept" => "application/json", 'Content-Type' => 'application/json' } }
+
+  describe 'creating a member' do
+    subject { post '/members', params: params.to_json, headers: headers }
+
+    context 'with valid params' do
+      let(:params) do
+        {
+          member: {
+            first_name: 'Sandi',
+            last_name: 'Metz',
+            url: 'http://www.example.com'
+          }
+        }
+      end
+
+      it 'returns the correct status code' do
+        subject
+        expect(response).to have_http_status(:success)
+      end
+    end
+
+    context 'with missing params' do
+      let(:params) { {} }
+
+      it 'returns the correct status code' do
+        subject
+        expect(response).not_to have_http_status(:success)
+      end
+    end
+  end
+
+  describe 'viewing all members' do
+    subject { get '/members', headers: headers }
+
+    it 'returns the correct status code' do
+      subject
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'returns an array' do
+      subject
+      expect(body).to be_an_instance_of(Array)
+    end
+  end
+
+  describe 'viewing a member' do
+    # context 'when member exists' do
```

Comment:
Should this be un-commented?

Reply:
Yeah I think that spec works! Super simple but proves that they implemented the route, and they could add more to it as they work on the feature